### PR TITLE
Update for SS5 compatibilty

### PR DESCRIPTION
**Pull request by Dennisprins93**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -14,19 +14,19 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": ">=8.1",
-        "silverstripe/cms": "^4@stable",
-        "silverstripe/vendor-plugin": "^1.0",
-        "silverstripe/display-logic": "^2.0",
-        "symbiote/silverstripe-gridfieldextensions": "^3.5"
+        "php": "^8.1",
+        "silverstripe/cms": "^5@stable",
```
Comment (renskorswagen) [inline composer.json]: silverstripe/framework: ^5
silverstripe/cms: ^5

File: composer.json [hunk 2]
```diff
@@ -14,19 +14,19 @@
...
+        "silverstripe/vendor-plugin": "^2.0",
```
Comment (renskorswagen) [inline composer.json]: is this necessary?

File: composer.json (hunk 1 above)
Comment (Dennisprins93) [reply composer.json]: Framework is already required via silverstripe/cms

File: composer.json (hunk 2 above)
Comment (Dennisprins93) [reply composer.json]: True. Removed.

File: composer.json [hunk 3]
```diff
@@ -14,19 +14,18 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "php": ">=8.1",
-        "silverstripe/cms": "^4@stable",
-        "silverstripe/vendor-plugin": "^1.0",
-        "silverstripe/display-logic": "^2.0",
-        "symbiote/silverstripe-gridfieldextensions": "^3.5"
+        "php": "^8.1",
+        "silverstripe/cms": "^5",
```
Comment (renskorswagen) [inline composer.json:18]: Add silverstripe/framework

**Pull request merged**